.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+on: 
+    pull_request:
+      branches: [ master ]
+    push:
+      branches: [ master ]
+name: Test
+jobs:
+    test:
```

Since this is the unit test, how about
```suggestion
    unit:
```

---

updated in current push